index.js

```diff
@@ -32,8 +32,12 @@ export class Cluster {
     body.append('file', file, file.name)
 
     const url = new URL('add', this.url)
-    url.searchParams.set('cid-version', 1)
-    url.searchParams.set('raw-leaves', true)
+    url.searchParams.set('cid-version', '1')
```

Comment:
`searchParams.set` takes a `string, string`, so making that explict here as i later compare a value from `searchParams.get` and it felt weird to pass a boolean in and assert a string on the other side.